agent.go

```diff
@@ -736,7 +736,7 @@ func (s *sandbox) listenToUdevEvents() {
 				if ch != nil && strings.HasPrefix(uEv.DevPath, filepath.Join(rootBusPath, devPCIAddress)) {
 					ch <- uEv.DevName
 					close(ch)
-					delete(s.deviceWatchers, uEv.DevName)
+					delete(s.deviceWatchers, devPCIAddress)
```

Comment:
I think actually there is no need to do the delete here,  it seems https://github.com/kata-containers/agent/blob/581bcf2404b23b53d0f1fbd9b833f7559fe2baa9/device.go#L162 will do this delete.

Reply:
yeah,  `agent/device.go#L162` deleted just when  time out happens.